packages/knip/src/plugins/github-actions/index.ts

```diff
@@ -26,13 +31,27 @@ const findGithubActionsDependencies: GenericPluginCallback = async (configFilePa
 
   if (!config) return [];
 
-  const scripts = getValuesByKeyDeep(config, 'run').filter((value): value is string => typeof value === 'string');
+  const scripts = getValuesByKeyDeep(config, 'run').filter(isString);
+
+  return [
+    ...getActionDependencies(),
+    ...getDependenciesFromScripts(scripts, {
+      cwd,
+      manifest,
+      knownGlobalsOnly: true,
+    }),
+  ];
+
+  function getActionDependencies() {
+    const isActionManifest = configFilePath.endsWith('action.yml') || configFilePath.endsWith('action.yaml');
```

Comment:
Recently I've refactored all `endsWith()` calls like this in plugins to `basename()`, would be great if that could be done here as well.

Reply:
Sure, done - https://github.com/webpro/knip/pull/432/commits/fb67cedcbd9441e19e344b204e1429c1b74664b7